SmartVision/WindowsForms/MissingPerson.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
```

Comment:
I believe that some using directives here are unnecessary. Not a major thing, but let's keep our code clean.

Reply:
Yes i kinda noticed that after commit. 👍